opensrp-chw/src/main/java/org/smartregister/chw/activity/PncMemberProfileActivity.java

```diff
@@ -401,6 +396,16 @@ protected void startFpRegister() {
         FpRegisterActivity.startFpRegistrationActivity(this, memberObject.getBaseEntityId(), memberObject.getDob(), CoreConstants.JSON_FORM.getFpRegistrationForm("Female"), FamilyPlanningConstants.ActivityPayload.REGISTRATION_PAYLOAD_TYPE);
     }
 
+    @Override
+    protected void startHivRegister() {
+        HivRegisterActivity.startHIVFormActivity(this, memberObject.getBaseEntityId(),org.smartregister.chw.util.Constants.JSON_FORM.getHivRegistration(),(new FormUtils()).getFormJsonFromRepositoryOrAssets(this, org.smartregister.chw.util.Constants.JSON_FORM.getHivRegistration()).toString());
+    }
+
+    @Override
+    protected void startTbRegister() {
+        TbRegisterActivity.startTbFormActivity(this, memberObject.getBaseEntityId(),org.smartregister.chw.util.Constants.JSON_FORM.getTbRegistration(),(new FormUtils()).getFormJsonFromRepositoryOrAssets(this, org.smartregister.chw.util.Constants.JSON_FORM.getTbRegistration()).toString());
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'org.smartregister.chw.util.Constants.JSON_FORM.getTbRegistration' due to existing static import 'org.smartregister.chw.util.Constants.JSON_FORM'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5590665)